pkg/agent/attestor/node/node.go

```diff
@@ -75,19 +77,19 @@ func (a *attestor) Attest(ctx context.Context) (res *AttestationResult, err erro
 		log.Info("Bundle loaded")
 	}
 
-	svid, key, err := a.loadSVID(ctx)
+	svid, key, reattestable, err := a.loadSVID(ctx)
 	if err != nil {
 		return nil, err
 	}
 
 	switch {
 	case svid == nil:
 		log.Info("SVID is not found. Starting node attestation")
-		svid, bundle, err = a.newSVID(ctx, key, bundle)
+		svid, bundle, reattestable, err = a.newSVID(ctx, key, bundle)
 		if err != nil {
 			return nil, err
 		}
-		log.WithField(telemetry.SPIFFEID, svid[0].URIs[0].String()).Info("Node attestation was successful")
+		log.WithField(telemetry.SPIFFEID, svid[0].URIs[0].String()).WithField("reattestable", reattestable).Info("Node attestation was successful")
```

Comment:
nit: for better or worse, we try to put these sort of logging constants in the pkg/common/telemetry package.

Reply:
done